ocs_ci/ocs/resources/cloud_manager.py

```diff
@@ -1,17 +1,22 @@
 import base64
 import json
 import logging
-import os
 from abc import ABC, abstractmethod
 from time import sleep
 
 import boto3
+import google.api_core.exceptions as GoogleExceptions
+from azure.core.exceptions import ResourceExistsError
+from azure.storage.blob import BlobServiceClient
```

Comment:
Please send an email regarding the new project requirements after the pr is merged.

Reply:
Done